spec/spec.tex

```diff
@@ -712,6 +712,30 @@ \subsubsection{Syntactic Shorthands}
       x <= d
 \end{lstlisting}
 
+To additionally aid readability, a \verb|when| statement whose conditional statement consists of a single line may be combined into a single line.
```

Comment:
I think it might be good to say something like "a conditional statement where the contents of the \verb|when| branch consist of a single line" to be consistent with the rest of the spec.